resolwe_bio/tests/test_chipseq.py

```diff
@@ -1,7 +1,7 @@
 # pylint: disable=missing-docstring
+import unittest
 from .utils import BioProcessTestCase
 
-
```

Comment:
Please, don't delete this line, it is required by [PEP8](https://www.python.org/dev/peps/pep-0008/#blank-lines).